### PR TITLE
update overlays to reference 1.24/stable charms

### PR DESCRIPTION
**Pull request by addyess**

### CODE REVIEW

File: overlays/equinix-overlay.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ machines:
     constraints: mem=32G
 applications:
   calico:
-    charm: cs:~containers/calico
+    charm: cs:~containers/calico-826
```
Comment (addyess) [inline overlays/equinix-overlay.yaml:10]: this is odd right?

File: overlays/ceph-fs.yaml [hunk 2]
```diff
@@ -10,12 +10,12 @@ applications:
       osd-devices: '32G,2'
       osd-journals: '8G,1'
   ceph-mon:
-    charm: ceph-mon
+    charm: cs:ceph-mon
```
[the remark this answers is not in the document]
Comment (addyess) [reply overlays/ceph-fs.yaml:13]: just got it with the revert--- i'm going back and fixing.

**Review by evilnick — APPROVED** [review]
:+1:

File: overlays/equinix-overlay.yaml (hunk 1 above)
Comment (kwmonroe) [inline overlays/equinix-overlay.yaml:10]: drop `cs:~containers/` here?

File: overlays/equinix-overlay.yaml [hunk 3]
```diff
@@ -7,7 +7,7 @@ machines:
     constraints: mem=32G
 applications:
   calico:
-    charm: cs:~containers/calico
+    charm: calico-826
```
Comment (evilnick) [inline overlays/equinix-overlay.yaml:10]: you should drop the -826

File: overlays/vault-pki-overlay.yaml [hunk 4]
```diff
@@ -6,10 +6,10 @@ applications:
     options:
       auto-generate-root-ca-cert: true
   percona-cluster:
-    charm: percona-cluster
+    charm: cs:percona-cluster
```
Comment (evilnick) [inline overlays/vault-pki-overlay.yaml:9]: no cs:

File: overlays/equinix-overlay.yaml [hunk 5]
```diff
@@ -125,20 +125,20 @@ relations:
 - - 'calico:etcd'
   - 'etcd:db'
 - - 'calico:cni'
-  - 'kubernetes-master:cni'
+  - 'kubernetes-control-plane:cni'
 - - 'calico:cni'
   - 'kubernetes-worker:cni'
-- - 'kubernetes-master:ceph-storage'
+- - 'kubernetes-control-plane:ceph-storage'
   - 'ceph-mon:admin'
```
Comment (kwmonroe) [inline overlays/equinix-overlay.yaml:132]:
```suggestion
```

We dropped the `ceph-storage` relation in https://github.com/charmed-kubernetes/charm-kubernetes-control-plane/pull/219

File: overlays/vault-pki-overlay.yaml (hunk 4 above)
Comment (kwmonroe) [inline overlays/vault-pki-overlay.yaml:9]:
```suggestion
    charm: percona-cluster
```

**Review by kwmonroe — COMMENTED** [review]
mostly lgtm, i think you need to drop the ceph-storage relation and one more `cs:` prefix.

**Review by kwmonroe — APPROVED** [review]
LGTM, probably need to hold this until 1.24 goes out.

**Pull request merged**